samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -60,32 +61,65 @@ public async Task ReassignWorkflow()
                     throw new Exception($"No submission found for {_config["studentAccount"]} in {assignmentId} for class {_config["classId"]}");
                 }
 
+                // Get submission outcomes
+                var submissionOutcomes = await Submission.GetSubmissionOutcomes(
+                    graphClient,
+                    _config["classId"],
+                    assignmentId,
+                    submissionId);
+
+                // Take the points outcome id
+                var pointsOutcomeId = submissionOutcomes.Value.Where(x => x.OdataType == "#microsoft.graph.educationPointsOutcome").Select(x => x.Id).FirstOrDefault();
+
+                // Create the points outcome body
+                var pointsOutcome = new EducationPointsOutcome
+                {
+                    OdataType = "#microsoft.graph.educationPointsOutcome",
+                    Points = new EducationAssignmentPointsGrade
+                    {
+                        OdataType = "#microsoft.graph.educationAssignmentPointsGrade",
+                        Points = 90
+                    }
+                };
+
+                // Update the submission points outcome
+                var returned = await Submission.PatchOutcomeAsync(
+                    graphClient,
+                    _config["classId"],
+                    assignmentId,
+                    submissionId,
+                    pointsOutcomeId,
+                    pointsOutcome);
+                Thread.Sleep(2000);
+                Console.WriteLine($"Points outcome updated: {pointsOutcome.Points.Points}");
```

Comment:
all these operations must be done with teacher account.
move all this block to line 113 after you change to teacher account

Reply:
moved code to teacher account.